### PR TITLE
docs: vibe coding macos app

### PR DESCRIPTION
**Pull request by mufassirkazi**

### CODE REVIEW

File: packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+
+The code still had to satisfy macOS. The API still had to satisfy GitHub. Those requirements did not go away. They just became more apparent through conversation rather than through stack traces and breakpoints. And instead of sitting, editing these files, I was describing problems and desired behavior. It was quite surprising, how little of this felt like coding. 
+
+
+If nothing else, building this app convinced me that evaluating coding agents only on greenfield generation misses most of the hard parts of software - permissions, lifecycle management, data modeling, and user-facing edge cases. The real test begins once the first version works and you start asking whether it actually fits into the world it is supposed to run in.
+
+Feel free to check out the full repository **[here](https://github.com/zwpaper/Ghlass)**. 
+
+You can also see the full trajectory of my interaction history with the agent **[here](https://app.getpochi.com/share/p-3b6489af950746878703552dc584256)**.
```
Comment (gyxlucy) [inline packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx]: link fails with 400 error. Please check and verify in browser incognito mode.

File: packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+title: 'Vibe Coding a macOS GitHub Client in Pochi'
+description: "Using a Coding Agent to build a macOS App in SwiftUI"
+image: "/og-image/vibe-coding-macos-app-cover-image.png"
+---
+
+# Vibe Coding a macOS GitHub Client in Pochi 
+
+
+<Callout title="TL;DR">
+   This app was built entirely through a single Pochi task. You can view the full prompt history and agent actions **[here](https://app.getpochi.com/share/p-3b6489af950746878703552dc5842561)**.
```
Comment (gyxlucy) [inline packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx:11]: Same link error here

File: packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx [hunk 3]
```diff
@@ -0,0 +1,238 @@
+---
+title: 'Vibe Coding a macOS GitHub Client in Pochi'
+description: "Using a Coding Agent to build a macOS App in SwiftUI"
+image: "/og-image/vibe-coding-macos-app-cover-image.png"
+---
+
+# Vibe Coding a macOS GitHub Client in Pochi
```
Comment (gyxlucy) [inline packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx:7]:
```suggestion
# Vibe Coding a MacOS GitHub Client in Pochi 
```

File: packages/docs/content/docs/developer-updates/index.mdx
Comment (gyxlucy) [inline packages/docs/content/docs/developer-updates/index.mdx:1]: This seems irrelevant to this tutorial. Please separate into a different PR.

File: packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx (hunk 1 above)
Comment (mufassirkazi) [reply packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx]: the link does work for me - even in icognito @gyxlucy

File: packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx (hunk 2 above)
Comment (mufassirkazi) [reply packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx:11]: link works for me - it has public access, can you share a screenshot?

File: packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx (hunk 3 above)
Comment (mufassirkazi) [reply packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx:7]: Screenshot 2026-01-28 at 10 04 55 PM


@gyxlucy

File: packages/docs/content/docs/developer-updates/index.mdx
Comment (mufassirkazi) [reply packages/docs/content/docs/developer-updates/index.mdx:1]: reverted back the changes.

File: packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx (hunk 2 above)
Comment (gyxlucy) [reply packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx:11]: This one actually works. But not the other one.

File: packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx (hunk 1 above)
Comment (gyxlucy) [reply packages/docs/content/docs/tutorials/vibe-coding-github-macos-app.mdx]: image

You are missing an 1 at the end of this link....

File: packages/docs/content/docs/developer-updates/index.mdx
Comment (gyxlucy) [inline packages/docs/content/docs/developer-updates/index.mdx:1]: Need to remove this file entirely from the PR.

**Review by gyxlucy — CHANGES_REQUESTED** [review]

**Review by gyxlucy — APPROVED** [review]
